### PR TITLE
add gosum to ensure correctly building

### PR DESCRIPTION
**Pull request by hermanbanken**
I tried to build after checking out the repo, but it failed due to an incompatibility with https://pkg.go.dev/google.golang.org/grpc@v1.35.0/resolver#ClientConn (which could probably be fixed in https://github.com/Jille/grpc-multi-resolver easily).
Fixed it by fixating the dependencies.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,24 @@
+module github.com/Jille/grpc-multi-resolver
```
Comment (Jille) [inline go.mod]: This should be Jille/raftadmin.

**Comment by Jille** [comment]
Groeten uit Nijmegen :)

**Comment by hermanbanken** [comment]
Groetjes terug 😉

**Comment by hermanbanken** [comment]
I didn't know how to correctly build the tool from a submodule without adding it to the dependencies in the cwd 😅 finally figured it out.

This only worked with the go.mod as added in this PR:
```bash
# to be also be able to also set GOOS and GOARCH
go build -o raftadmin github.com/Jille/raftadmin/cmd/raftadmin
```

This works also, but doesn't offer GOOS/GOARCH.
```bash
# somehow install accepts @fork while build doesn't
go install github.com/hermanbanken/raftadmin/cmd/raftadmin@fork
```

**Comment by Jille** [comment]
Would you mind running `go mod tidy`? Then I'll squash and merge.

**Pull request merged**